src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -1,554 +0,0 @@
-<!--
```

Comment:
As spoken yesterday, I didn't have that at the time, I've broken down things by version now so that 5.0 and earlier use the old version, and 6.0 and onwards use the new version